.gitignore

```diff
@@ -1 +1,2 @@
 build
+**/.DS_Store
```

Comment:
Delete .DS_Store

Reply:
This will be a problem for everyone using a mac (it gets auto-created as an index file). Recommend keeping.

Reply:
Ah, I completely misunderstood. This is fine.